.github/workflows/snyk-pr.yml

```diff
@@ -0,0 +1,18 @@
+# This action runs every day at 6 AM and on every push
+# If the branch it's running on is main then it will run snyk monitor (reports vulnerabilities to snyk.io)
+# Otherwise it will run snyk test
+name: Snyk
+
+on:
+  pull_request:
+    branches:    
+      - main
```

Comment:
Does this restriction add much? I sometimes (initially) target PRs against other branches if they build on top of each other, before merging them in sequence - this would mean the check doesn't run at first. It's probably only a minor inconvenience.

Reply:
and as discussed it'd be good to check out a `paths` restriction to see if we can restrict to when manifest files are changed

Reply:
and as discussed it'd be good to check out a `paths` restriction to see if we can restrict to when manifest files are changed